cmd/ddev/cmd/version.go

```diff
@@ -14,21 +15,31 @@ var versionCmd = &cobra.Command{
 	Short: "print ddev version and component versions",
 	Long:  `Display the version of this ddev binary and its components.`,
 	Run: func(cmd *cobra.Command, args []string) {
-		table := uitable.New()
-		table.MaxColWidth = 200
-
-		table.AddRow("cli:", version.DdevVersion)
-		table.AddRow("web:", version.WebImg+":"+version.WebTag)
-		table.AddRow("db:", version.DBImg+":"+version.DBTag)
-		table.AddRow("dba:", version.DBAImg+":"+version.DBATag)
-		table.AddRow("router:", version.RouterImage+":"+version.RouterTag)
-		table.AddRow("commit:", version.COMMIT)
-		table.AddRow("build info:", version.BUILDINFO)
-
-		fmt.Println(table)
+		if len(args) != 0 {
+			util.Failed("invalid arguments to version command: %v", args)
```

Comment:
I would prefer to see Usage be output with this, similar to how we do with start: https://github.com/drud/ddev/blob/master/cmd/ddev/cmd/start.go#L29